test/pay/webhooks/delegator_test.rb

```diff
@@ -40,6 +41,13 @@ def call(event)
     assert processor.success
   end
 
+  test "can subscribe with a string" do
```

Comment:
Can't say I'm thrilled with this test. Basically taking advantage of Ruby's pass-by-ref and relying on the event object being mutated...

This is also hard to replicate the Rails boot process as the processor class here has already been defined. Going to assume `NotificationAdapter` isn't actually invoked to run the processor until app boot is done (which should be the case?).